packages/react-components/lib/workcells/workcell-table.tsx

```diff
@@ -1,4 +1,12 @@
-import { makeStyles, Table, TableBody, TableCell, TableHead, TableRow } from '@material-ui/core';
+import {
```

Comment:
![image](https://user-images.githubusercontent.com/45189696/137829977-c3dc3511-51b3-4be7-8c79-023f7f21c739.png)

I think it looks better if the dispensers and ingestors table columns are aligned.